config/sync/core.extension.yml

```diff
@@ -130,7 +130,6 @@ module:
   responsive_image: 0
   scheduled_transitions: 0
   scheduler: 0
-  scheduler_content_moderation_integration: 0
```

Comment:
Wasn't listed on NIDirect so turned it off here too. Can always re-enable later if needed